### PR TITLE
Update Grpc.AspNetCore.Server version

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This should fix https://devdiv.visualstudio.com/DevDiv/_workitems/edit/824934?src=WorkItemMention&src-action=artifact_link and https://devdiv.visualstudio.com/DevDiv/_workitems/edit/819262?src=WorkItemMention&src-action=artifact_link

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -174,7 +174,7 @@
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
     <GoogleProtobufPackageVersion>3.6.1</GoogleProtobufPackageVersion>
-    <GrpcAspNetCoreServerPackageVersion>0.1.19-pre1</GrpcAspNetCoreServerPackageVersion>
+    <GrpcAspNetCoreServerPackageVersion>0.1.19-pre2</GrpcAspNetCoreServerPackageVersion>
```
Comment (JunTaoLuo) [inline eng/Versions.props]: This is an unlisted nightly build hosted on nuget.org

**Review by JamesNK — APPROVED** [review]

File: eng/Versions.props (hunk 1 above)
Comment (dougbu) [reply eng/Versions.props]: Fine from my perspective. Check with @Eilon about approving the new version if you haven't already.

**Review by dougbu — APPROVED** [review]

File: eng/Versions.props (hunk 1 above)
Comment (JunTaoLuo) [reply eng/Versions.props]: @eilon, can you comment? Waiting for your green light before I merge.

Comment (JamesNK) [reply eng/Versions.props]: To be clear: this version update is to get the template successfully running with nightly builds of the .NET SDK.

Prior to preview 4 GA the template will be updated to a publicly list version of Grpc.AspNetCore.Server.

**Review by Eilon — APPROVED** [review]

**Pull request merged**